pages/api/extension/trigger.ts

```diff
@@ -20,11 +21,11 @@ export default async function triggeHandler(req: NextApiRequest, res: NextApiRes
 		console.error("[extension/triggeHandler] Error getting user token", err);
 		return null;
 	});
-    if (!user?.email) {
+    if (!user || !user.email) {
```

Comment:
Did the optional chaining not handle some situation?

Reply:
This was a bad diff, hadn't merged main into this branch